tests/unit/clients/sqlalchemy_tests.py

```diff
@@ -39,6 +39,7 @@ def test_credentials(self, create_engine_mock):
             {
                 "database.url": "postgresql://fizz:buzz@localhost:9000/db",
                 "database.credentials_secret": "secret/sql/account",
+                "database.pool_recycle": 60,
```

Comment:
the value on the right should be a string to match how a config dictionary will work.

Reply:
I think this is how it will be. 

https://github.snooguts.net/reddit/puppet/blob/919fead76757608bb5e26848a8a8616892a75cec/modules/reddit_thing_server/templates/thing.ini.erb#L31

Reply:
No, it'll be a string because `configparser` only deals in strings. The `config.RawConfig` type is `Dict[str, str]`.